### PR TITLE
[CET-9710] Initiative due date discrepancy between Cortex and Backstage

### PR DESCRIPTION
**Pull request by PavelZurek**
## Change description

This partially reverts #183. Namely converting the Initiatives target date to local TZ.

## Type of change

- [x] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)
- [ ] Improvement (improves codebase without changing functionality)

## Screenshots

Before this change, UTC-7 (shifted)
image

After this change, UTC-7 (not shifted)
image

After this change, UTC+2 (not shifted)
image

### Development

- [ ] The changelog has been updated as appropriate
- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

**Review by browniefed — APPROVED** [review]

**Comment by PavelZurek** [comment]
Thanks @igorRoog , I fixed the timezone to be resolved correctly, the values displayed are the same for `moment(targetDate).local()` and `moment.utc(targetDate)` in this case due to missing TZ in the `targetDate`.

**Comment by igorRoog** [comment]
I would love to see some printscreens we display same date in brain-app and backstage-plugin after this change.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 2.11.4
+
+- Revert converting to local timezone for Initiative target date
```
Comment (cristinabuenahora) [inline CHANGELOG.md:5]: Let's change this to: 

```suggestion
- Update how we display Initiative target date
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+
 ### 2.11.3
 
 - Fix for opening initiative filtered by email owner who doesn't exist
```
Comment (cristinabuenahora) [inline CHANGELOG.md:9]:
```suggestion
- Fix for opening Initiative filtered by email owner who doesn't exist
```

**Review by cristinabuenahora — APPROVED** [review]

**Pull request merged**